specifications/api/swagger.yaml

```diff
@@ -2521,6 +2534,21 @@ components:
        quantity: *assetQuantity
        fingerprint: *assetFingerprint
 
+    PlutusScriptInfo: &PlutusScriptInfo
+      type: object
+      required:
+        - language_version
+        - script_hash
```

Comment:
yes, it is the same script hash. for plutus we could of course at this moment (when we detect script in TxOut) include all binary of plutus file but I decided right now to add just hash of it as a representative of it. When in the future we will add premium plutus support (I hope we do) then we could add better summary of the script or just add raw binary. In case of native script we have just script and as we support native scripts in our code base I use it